### ISSUE TITLE
Resolving the failing `grackle_general` test

### ISSUE THREAD

**Issue by mabruzzo** [enzo-project/enzo-e#289]
Today I did some investigating into why the continuous integration started reporting test failures for a bunch of PRs for the `grackle_general` test.

After digging around, I ultimately confirmed locally that this test ultimately started failing as a result of grackle-project/grackle#126. That PR changed how Grackle performs redshift interpolation of tabulated data. (Note I am aware that the failing test is a non-cosmological simulation, but it the failure may now be arising because of some other incidental changes). In other words, the reference results were recorded while using an older version of Grackle and newer versions give slightly different results.

While the "quick" workaround for this problem is to simply alter the hard-coded reference values used by the test, the "optimal" solution is already in a pending PR: #267.

- the problem with the "quick" solution is that this exact same problem will crop again in the future when other algorithmic/numeric improvements are made to Grackle
- In contrast, PR #267, moves the grackle tests to the answer testing framework. The new tests will always determine the reference results and the actual results with the same version of Grackle. (I know that there have historically been issues with maintaining API compatibility between Enzo-E and different versions of Grackle, but that will be addressed in a PR that I'm adding finishing touches to)

In short, it would be helpful to merge PR #267 ASAP to address the failing test. That PR currently has 1 approval. It needs one other person to review it
